### PR TITLE
Try in Gitpod button on Website links to Gitpod VS Code announcement

### LINKED ISSUE #148: Try in Gitpod button on Website links to Gitpod VS Code announcement
While creating the Theia Blueprint article for the EF April newsletter I noticed that the "Try in Gitpod" button on https://theia-ide.org/ prompts new users with this message:
image 

Since the button is shown very prominently on the Theia landing page and promises to let me try Theia in Github this is rather unexpected. What do you think, @marcdumais-work, @svenefftinge, @vince-fugnitto ? 
Can we maybe add a different button like "Try in OpenShift" as an alternative? This topic is rather urgent since the newsletter will be published next week imho and will hopefully attract visitors to the Theia webpage.

### PR DESCRIPTION
**Pull request by jfaltermeier**
* remove "Try in Gitpod" button for now
* closes #148

Signed-off-by: Johannes Faltermeier <jfaltermeier@eclipsesource.com>

Contributed on behalf of STMicroelectronics

### CODE REVIEW

File: src/components/index/Header.js [hunk 1]
```diff
@@ -91,7 +91,6 @@ const Header = () => (
                     </h2>
 
                     <a className="btn" href="https://github.com/eclipse-theia/theia" target="_blank" rel="noopener noreferrer">View on GitHub</a>
-                    <a className="btn btn--cta" href="https://gitpod.io/#https://github.com/eclipse-theia/theia" target="_blank" rel="noopener">Try in Gitpod &nbsp;&nbsp;&rarr;</a>
                     <a className="btn btn--cta" href="/docs/blueprint_download/" rel="noopener">Try on Desktop &nbsp;&nbsp;&rarr;</a>
```
Comment (koegel) [inline src/components/index/Header.js:94]: I suggest to rename the button to "Try now &nbsp;&nbsp;&rarr;"

Comment (jfaltermeier) [reply src/components/index/Header.js]: done

File: src/components/index/Header.js [hunk 2]
```diff
@@ -91,8 +91,7 @@ const Header = () => (
                     </h2>
 
                     <a className="btn" href="https://github.com/eclipse-theia/theia" target="_blank" rel="noopener noreferrer">View on GitHub</a>
-                    <a className="btn btn--cta" href="https://gitpod.io/#https://github.com/eclipse-theia/theia" target="_blank" rel="noopener">Try in Gitpod &nbsp;&nbsp;&rarr;</a>
-                    <a className="btn btn--cta" href="/docs/blueprint_download/" rel="noopener">Try on Desktop &nbsp;&nbsp;&rarr;</a>
+                    <a className="btn btn--cta" href="/docs/blueprint_download/" rel="noopener">Try now &nbsp;&nbsp;&rarr;</a>
```
Comment (JonasHelming) [inline src/components/index/Header.js:94]: What is the reason to rename to "try now"?  "try on desktop" indicated that there are other options (i.e. in the browser) so I would prefer it.

Comment (koegel) [reply src/components/index/Header.js:94]: I asked @jfaltermeier to change exactly that :). I think it looks strange with the text "Try on Desktop" if there is no other option, but I do not have my heart set on it.
See preview: https://deploy-preview-151--romantic-mcclintock-8d974f.netlify.app/

Comment (JonasHelming) [reply src/components/index/Header.js:94]: I see :-)

Due to https://github.com/eclipse-theia/theia-website/issues/148#issuecomment-823373640 I would be in favor of "try on desktop", but no strong opinion

Comment (koegel) [reply src/components/index/Header.js:94]: I am in favor of changing it back whenever we have a second button again and to leave it as is for now.

**Review by JonasHelming — APPROVED** [review]

**Pull request merged**